lms/security.py

```diff
@@ -132,8 +216,17 @@ def _get_lti_user(request):
     If the request doesn't contain either valid LTI launch params or a valid
     bearer token then return ``None``.
 
+    :param: from_identity In case we are using this function directly on a security policy
```

Comment:
This allows all the code that does `request.lti_user` to use the LTIUser created by `LTILaunchesSecurityPolicy`